docs/tutorial.md

```diff
@@ -386,8 +376,7 @@ spec:
           value: "spec.template.spec.containers[0].image"
 ```
 
-The above `Pipeline` is referencing a `Task` called `deploy-using-kubectl` which
-can be found here:
+The above `Pipeline` is referencing a `Task` called `deploy-using-kubectl` defined as follows:
```

Comment:
```suggestion
The above `Pipeline` is referencing a `Task` called `deploy-using-kubectl`, which is defined below:
```

Reply:
The two wordings convey the same meaning.